### PR TITLE
Test nefertari_starter scaffold

### PR DESCRIPTION
**Pull request by postatum**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,3 +7,12 @@ env:
 install:
   - pip install tox
 script: tox
+services:
+  - mongodb
+before_install:
+  - travis_retry curl -O https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.7.5.deb && sudo dpkg -i --force-confnew --force-confdef elasticsearch-1.7.5.deb
```
Comment (jstoiko) [inline .travis.yml]: why not use ES 2.x (current stable version)?

File: setup.py [hunk 2]
```diff
@@ -46,6 +46,7 @@
     [console_scripts]
         nefertari.index = nefertari.scripts.es:main
         nefertari.post2api = nefertari.scripts.post2api:main
+        nefertari.scaffold_test = nefertari.scripts.scaffold_test:main
```
Comment (jstoiko) [inline setup.py]: why is it necessary to expose `scaffold_test` in console_scripts? If not, I would remove this line.

File: .travis.yml (hunk 1 above)
Comment (postatum) [inline .travis.yml]: Because ES2 support in our frameworks is not released yet.

File: setup.py (hunk 2 above)
Comment (postatum) [inline setup.py]: Removed.

**Pull request merged**